tests/lib/rules/indent.js

```diff
@@ -1504,7 +1504,7 @@ ruleTester.run("indent", rule, {
             "{\n" +
             "      bar();\n" +
             "}",
-            options: [2, {FunctionDeclaration: {body: 3}}] // FIXME: what is the default for `parameters`?
+            options: [2, {FunctionDeclaration: {body: 3}}]
```

Comment:
(This removal isn't actually related to this PR; I forgot to remove the comments in #7076.)